src/chart/SunburstChart.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import { scaleLinear } from 'victory-vendor/d3-scale';
+import { Surface } from '../container/Surface';
+import { Layer } from '../container/Layer';
+import { Sector } from '../shape/Sector';
+import { Text } from '../component/Text';
+
+export interface SunburstData {
+  name: string;
+  value?: number;
+  fill?: string;
+  children?: SunburstData[];
+}
+
+interface TextOptions {
+  fontFamily?: string;
+  fontWeight?: string;
+  paintOrder?: string;
+  stroke?: string;
+  fill?: string;
+}
+
+export interface SunburstChartProps {
+  data?: SunburstData;
+  width?: number;
+  height?: number;
+  padding?: number;
+  ringPadding?: number;
+  innerRadius?: number;
+  children?: React.ReactNode;
+  fill?: string;
+  stroke?: string;
+  textOptions?: TextOptions;
+}
+
+interface DrawArcOptions {
+  r: number;
+  innerR: number;
+  initialAngle: number;
+  childColor?: string;
+}
+
+const defaultTextProps = {
+  fontFamily: 'sans-serif',
+  fontWeight: 'bold',
+  paintOrder: 'stroke fill',
+  stroke: '#FFF',
+  fill: 'black',
+};
+
+function maxDepth(node: SunburstData): number {
+  if (!node.children || node.children.length === 0) return 1;
+
+  // Calculate depth for each child and find the maximum
+  const childDepths = node.children.map(d => maxDepth(d));
+  return 1 + Math.max(...childDepths);
+}
+
+function polarToCartesian(r: number, angleInDegrees: number): number[] {
```

Comment:
This seems pretty generic. We likely already have this implemented elsewhere. Please check, or move into a generic file such as PolarUtils or CartesianUtils